extensions/algolia-indexer/index.js

```diff
@@ -117,21 +119,11 @@ function register({
     console.log(chalk.green('New records:' + totalObjectsToAdd))
 
     totalObjectsToAdd === 0 && totalObjectsToUpdate === 0 && console.log(chalk.green('No changes made'))
+  })
 
-    try {
-      let recordCount = 0
-
-      await index.browseObjects({
-        query: '',
-        batch: batch => {
-          recordCount += batch.length
-        }
-      })
-
-      console.log(chalk.green('Total records:', recordCount))
-    } catch (err) {
-      logger.error(JSON.stringify(err))
-    }
+  process.on('exit', () => {
+    httpAgent.destroy()
```

Comment:
Isn't it better to change the flag from keepAlive to false? 

Does this get triggered both on success and error?

Reply:
It should get triggered when the Node.js process is about to exit, regardless of the reason for the exit.